apps/test/unit/templates/studioHomepages/TeacherHomepageTest.jsx

```diff
@@ -0,0 +1,400 @@
+import {render, screen} from '@testing-library/react';
```

Comment:
Refactored test to RTL/Jest because that was actually easier. (and renamed it to `jsx`)